build.gradle

```diff
@@ -57,6 +74,51 @@ dependencies {
     testCompile 'junit:junit:4.12'
 }
 
+task cloneParity(type: com.palominolabs.gradle.task.git.clone.GitCloneTask) {
+  dir = file("$buildDir/parity")
+  uri = 'https://github.com/RightMesh/vagrant-eth-env.git'
+  treeish = 'origin/master' // a commit hash, or tag name, or branch name (with remote prefix, like 'origin/master')
```

Comment:
Does this comment need to be here?

Also - do you always want to be linked to `HEAD` of `origin/master`? Would a fixed commit be safer so it doesn't break on you unexpectedly?

Reply:
Yes that's probably best - tbh I just used the default that already existed for this. Will update to the most recent commit.